app/policies/team_policy.rb

```diff
@@ -23,15 +25,15 @@ def edit?
   end
 
   def destroy?
-    admin?
+    in_team? || admin?
```

Comment:
Wanted to acknowledge that this seems scary but it follows this:

https://docs.google.com/document/d/17gh18BoeJ84ZZKX_z78xuDGHk4Lko9AfcvtctOqT-y4/edit#

Reply:
yeah, I made a comment in that doc... if a team is deleted, do we delete all of the apps associated with that team? And are teams really deleted all that often that we need to make it easy for partners?

Reply:
I agree!